src/cards/__integ__/sticky-header.test.ts

```diff
@@ -47,7 +47,7 @@ describe('Cards Sticky Header', () => {
   const toggleVerticalOffsetBtn = '#toggle-vertical-offset-btn';
   const overflowParentPageHeight = 300;
   const verticalOffset = 50;
-  const containerBorder = 1;
+  const containerBorder = CONTAINER_ROOT_BORDER;
```

Comment:
This const seems a bit redundant. What's the different between border and root border?

Reply:
The way this variable is used is also a bit odd since we add and subtract 0?

Reply:
I think one was for setting the default position in the overflow container, and the other was an offset correction for the table itself, but I agree it seems a bit redundant. I removed both, as well as the code that required the correction in the next revision.